pkg/prometheus/servicemonitor.go

```diff
@@ -23,8 +23,20 @@ func newMonitoringClient() (*monitoringclient.Clientset, error) {
 	return monitoringClient, err
 }
 
+func ServiceMonitorExists(sm *monitoringv1.ServiceMonitor) (*monitoringv1.ServiceMonitor, error) {
+	serviceMonitorClient, err := newMonitoringClient()
```

Comment:
newMonitoringClient is in camelcase. change the function name and the following declarations.

Reply:
This is required.